### DISCUSSION TITLE
Cant upload documents

### DISCUSSION THREAD

**Discussion by horphi0815** [jonaswinkler/paperless-ng#826] [Support] [answered]
Hello together,
i cant upload documents. Its a new blank paperless instance.
HTTP-Fehler: 500 Internal Server Error

Where can i see the logs, to identify the root cause?

BR 

Phil

**Comment by jonaswinkler** [comment, marked as answer]
It's pretty hard to help since you did not provide any information about your setup.

500 Internal Server Error most likely indicates that redis is not configured properly.

**Reply by horphi0815** [reply]
Ok i have no redis setup
Maybe thats the fault...

**Reply by horphi0815** [reply]
Thank you, thats it. Redis IP was missisg :-)

BR,
Phil

**Reply by manuel-simon-relative** [reply]
How do you have to proceed here? Does anyone have any precise information?

**Reply by manuel-simon-relative** [reply]
I have redis running in separate docker and password and port entered in paperless-docker.

**Reply by devPeete** [reply]
> How do you have to proceed here? Does anyone have any precise information?

I Suggest to use one of the Docker Compose files, they are fully configured.
You just have to make sure, that all containers are up and running (_sudo docker ps -a_ should list all containers as running and not e.g. stopped). Furthermore I suggest to switch to the new [Paperless-NGX repository ](https://github.com/paperless-ngx/paperless-ngx)as this fork is most likely dead.

**Reply by manuel-simon-relative** [reply]
Yes, I use paperless-ngx. The error is still the same on my Synology NAS. Thank you.
